lib/src/components/NxToggle/NxToggle.scss

```diff
@@ -31,12 +32,22 @@
 
   &:focus-within {
     .nx-toggle__control {
-      border-color: var(--nx-color-interactive-border-focus);
-      filter: drop-shadow(var(--nx-drop-shadow-focus));
-
       &:hover {
         border-color: var(--nx-swatch-grey-05);
       }
+
+      &::after{
+        border: 1px solid var(--nx-color-interactive-border-focus);
+        border-radius: 16px;
+        box-shadow: var(--nx-box-shadow-focus);
+        content: '';
+        display: block;
+        height: 30px;
+        position:absolute;
+        left: -4px;
+        top: -4px;
+        width: 50px;
```

Comment:
Should we turn the `height` and `width` of the control into local scss variables and derive the focus border sizes from it?

Reply:
I think it's a good idea, but I think would it make sense to do it under a different JIRA ticket and use variables for focus borders of radios and checkboxes as well?

Reply:
> I think it's a good idea, but I think would it make sense to do it under a different JIRA ticket and use variables for focus borders of radios and checkboxes as well?

Sure.  Put in a tech debt ticket and link it here.

Reply:
> > I think it's a good idea, but I think would it make sense to do it under a different JIRA ticket and use variables for focus borders of radios and checkboxes as well?
> 
> Sure. Put in a tech debt ticket and link it here.

Tech Debt ticket: https://issues.sonatype.org/browse/RSC-620